deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -122,7 +122,7 @@
     'tnr': make_scorer(true_negative_rate_metric, averaging_method='binary'),
     'jaccard': make_scorer(jaccard_score, zero_division=0),
     'roc_auc': get_scorer('roc_auc'),
-    'neg_log_loss': get_scorer('neg_log_loss')
+    'neg_log_loss': make_scorer(log_loss, greater_is_better=False, needs_proba=True, labels=[0, 1])
```

Comment:
is it still `neg_log_loss` after the `greater_is_better=False`?

Reply:
+ I am not sure if log_loss is the same as neg_log_loss, If not - its bad:
I think can create weird behaviour in some of our checks, we require that scorers also follow the convention that greater is better. even written in our docs

Reply:
I think we need to wrap the log_loss in lambda:
```
def neg_log_loss(y, y_pred, **kwargs):
  return -log_loss(y, y, y_pred, **kwargs)
```

Reply:
this is how sklearn defines neg_log_loss
you can see it by printing get_scorer('neg_log_loss')
I only added the labels